ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -382,3 +382,43 @@ if get_option('enable-openvino')
     include_directories: include_directories('.')
   )
 endif
+
+if get_option('enable-mediapipe')
+
+  cmd = run_command('sh', '-c', 'echo $MEDIAPIPE_HOME')
+  MEDIAPIPE_HOME = cmd.stdout().strip()
+
+  if run_command('[', '-d', MEDIAPIPE_HOME, ']').returncode() == 0
+    message('MEDIAPIPE_HOME: ' + MEDIAPIPE_HOME)
+    mediapipe_incdir = include_directories(
+      MEDIAPIPE_HOME,
+      join_paths(MEDIAPIPE_HOME, 'bazel-bin'),
+      join_paths(MEDIAPIPE_HOME, 'bazel-mediapipe/external/eigen_archive'),
+      join_paths(MEDIAPIPE_HOME, 'bazel-mediapipe/external/com_google_absl'),
+    )
+
+    cmd = run_command('sh', '-c', '../../../tools/development/gen_mediapipe_libs.sh ' + nnstreamer_libdir)
```

Comment:
I think you can utilize `meson.source_root()` to access `tools` directory

Reply:
That's what I need! Thanks!!